lib/payday/pdf_renderer.rb

```diff
@@ -302,7 +302,7 @@ def self.bold_cell(pdf, text, options = {})
     def self.number_to_currency(number, invoice)
       currency = Money::Currency.wrap(invoice_or_default(invoice, :currency))
       number *= currency.subunit_to_unit
-      number = number.round unless Money.infinite_precision
+      number = number.round unless Money.default_infinite_precision
```

Comment:
Did you check that this is the same valjue?

Reply:
```
[DEPRECATION] `Money.infinite_precision` is deprecated - use `Money.default_infinite_precision` instead
```